docs/rundev.md

```diff
@@ -0,0 +1,90 @@
+# Running Dataverse development snapshots
+
+Images on Docker Hub are meant for production usage or quick demos.
+When developing Dataverse, testing a new feature not yet shipped in a release or
+running integration tests you have a need to deploy all moving parts into
+a (more or less) ephemeral environment.
+
+**Be warned:** deployment times of Dataverse are a nightmare as of writing (Sept. 2019).
+For any change to the codebase, you most likely will have to wait about five
+to ten minutes to redeploy from compilation to reload of webpage.
+
+This is due to a really big WAR file (which needs to be loaded into the cluster)
+and being stuck on old technology in combination with a bloated monolith.
```

Comment:
"bloated" is a bit harsh. How about "gigantic"? 😄

Reply:
LOL. I choosed bloated because of things like Spring included in a Java EE app... :wink:

Reply:
Done in c874e78.